app/discord.server.ts

```diff
@@ -0,0 +1,46 @@
+import crypto from "node:crypto";
+import DiscordOauth2 from "discord-oauth2";
+
+const DISCORD_REDIRECT_URI =
+  process.env.NODE_ENV === "production"
```

Comment:
**Thought (non-blocking)**: If we do checks like this in many places, we might want to adopt something like the codesee_metaconfig.json file for storing non-sensitive constants that are env specific.

Reply:
Good idea. This is the only place where we're doing this so far but I'll keep an eye on it.